File: CS/CustomItemExtension/IExtensionModule.cs
using DevExpress.DashboardWin;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DevExpresss.DashboardWin.CustomItemExtension {
    public interface IExtensionModule {
        void AttachViewer(DashboardViewer viewer);
        void DetachViewer();
        void AttachDesigner(DashboardDesigner designer);
        void DetachDesigner();
    }
}

File: VB/CustomItemExtension/IExtensionModule.vb
Imports DevExpress.DashboardWin
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DevExpress.DashboardWin.CustomItemExtension
	Public Interface IExtensionModule
		Sub AttachViewer(ByVal viewer As DashboardViewer)
		Sub DetachViewer()
		Sub AttachDesigner(ByVal designer As DashboardDesigner)
		Sub DetachDesigner()
	End Interface
End Namespace
